frontend/src/components/experimentRuns/ExperimentRunValidation.tsx

```diff
@@ -63,38 +63,32 @@ function ExperimentRunValidation({ experimentRunName }: ExperimentRunValidationP
 	const setPassed = useCallback(
 		(lane: LaneInfo) => {
 			setIsValidationInProgress(true)
-			dispatch(setRunLaneValidationStatus(lane, ValidationStatus.PASSED))
-				.finally(() => {
-          lane.datasets.map((dataset) => {dispatch(get(dataset.datasetID))})
-          setIsValidationInProgress(false)
-        }
-      )
+			dispatch(setRunLaneValidationStatus(lane, ValidationStatus.PASSED)).finally(() => {
+				Promise.allSettled(lane.datasets.map((dataset) => dispatch(get(dataset.datasetID)))).finally(() =>
```

Comment:
We don't need to wait for it ... it'll update on its own time ... if we wait for all async calls it kind of defeat the purpose...

Reply:
Fine.